README.md

```diff
@@ -1 +1,63 @@
-# Seminar--Capstone-
\ No newline at end of file
+# Seminar (Capstone Project)
+
+> This project contains home and about pages of a behance design template to imitate a prototype of annual seminar on trading.
+> The features in this project were created using javascript, html and css.
+>Javascript features (object) is also inculcated.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://yussufob.github.io/Seminar--Capstone-/)
+
+### Prerequisites
+- HTML, CSS & JAVASCRIPT
+
+### Setup
+- linters for HTML, CSS and Javascript
+- Standard Accessibility tools (https://www.accessi.org/)
+
+### Run tests
+- Open your terminal, move to the directory where you want to clone the app (i.e cd Documents) 
+- Enter git clone git@github.com:YussufOB/Seminar-Capstone-.git
+- Change to the directory you just clone (i.e cd Seminar-Capstone-)
+- Open index.html from the list of files in your left handside
+- Right click on your opened index.html file and launch the live server to preview
+- Include any possible changes and run "git add ." 
+- Commit the changes with a descriptive message (i.e xyz features added) 
+- Push the changes to changes to the repository (i.e git push)
+
+### Deployment
+- This app is deployed using Github pages (https://pages.github.com/)
+
+## Author
+
+👤 **Bashir Yussuf Olamilekan (Bash)**
+
+- GitHub: [@YussuOB](https://github.com/YussufOB)
+- Twitter: [@twitterhandle](https://twitter.com/_ybash)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/yussufOB)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to all feature person and the brands that their logo was used.
+- Inspiration comes from my interest in technical analysis and support for the Cotton Candy work.
+
```

Comment:
- [ ] You should give credit to [Cindy Shin in Behance](https://www.behance.net/adagio07) for the Original design idea, as stated in the project requirement. 
> The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.